components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -70,10 +71,11 @@ export default function Preferences() {
                         );
                     }
                 }
+                // Reset creationError to avoid displaying the error message and toast the success message
+                setCreationError(undefined);
```

Comment:
I probably would have gone with a toast here for the error instead of an inline one.